java/src/test/java/ai/rapids/cudf/BinaryOpTest.java

```diff
@@ -992,4 +993,91 @@ public void testOr() {
       }
     }
   }
+
+  @Test
+  public void testShiftLeft() {
+    try (ColumnVector icv = ColumnVector.fromBoxedInts(INTS_2);
```

Comment:
`INTS_2` doesn't contain very many interesting values for this test.  None of the values are negative, zero, or very large.  This applies to the other tests as well.